src/scripts/blah_common_submit_functions.sh

```diff
@@ -314,6 +314,10 @@ function bls_parse_submit_options ()
       N) bls_opt_hostsmpsize="$OPTARG";;
       z) bls_opt_wholenodes="$OPTARG";;
       h) bls_opt_hostnumber="$OPTARG";;
+      g) bls_opt_gpunumber="$OPTARG";;
+      m) bls_opt_gpumode="$OPTARG";;
```

Comment:
This is an issue, there is already a `-m` option (`bls_opt_req_mem`)

Reply:
I will change -m to -G.